### PR TITLE
Migrate  old product categories to new categories 

### PR DESCRIPTION
**Pull request by StupidCodeFactory**
## Description 

migrate all categories to new categories 

## notes 
I was not sure about the "Alarms" old category so I've put to map to "Others" happy to correct it whenever

### CODE REVIEW

**Comment by StupidCodeFactory** [comment]
[Merge 'old' categories](https://trello.com/c/ZbF9CVqp/801-merge-old-categories)

**Review by slorek — APPROVED** [review]
Await @tobyhughes approval before merging.

File: db/migrate/20201109154238_fix_old_category.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+       "White Goods",
+       "Small Electronics"] => "Electrical appliances and equipment",
+      ["Baby/Children's Products"] => "Childcare articles and children's equipment",
+      ["Other Product Type"] => "Other",
+      ["Furniture & Furnishings"] => "Furniture",
+      ["Clothing (inc. baby)"] => "Clothing, textiles and fashion items",
+      ["Personal Protective Equipment (PPE)."] => "Personal protective equipment (PPE)",
+      ["Measuring Instruments (inc. pressure)"] => "Measuring instruments",
+      %w[Lasers] => "Laser pointers",
+      %w[Alarms] => "Other"
```
Comment (slorek) [inline db/migrate/20201109154238_fix_old_category.rb:14]: I think there's some more variants in the example attached to the Trello card.

Comment (StupidCodeFactory) [reply db/migrate/20201109154238_fix_old_category.rb:14]: This was me going into the DB and checking product against the up to date categories. I think this is all really.

**Pull request merged**